### PR TITLE
feat: add devnet 10

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: ansible/inventories/devnet-10/group_vars/all/all.yaml [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+ethereum_genesis_mnemonic: "{{ secret_genesis_mnemonic }}"
+ethereum_genesis_generator_config_files:
+  cl/config.yaml: "{{ lookup('ansible.builtin.url', 'https://raw.githubusercontent.com/ethpandaops/ethereum-genesis-generator/v{{ethereum_genesis_generator_version}}/config-example/cl/config.yaml', split_lines=false) }}" # noqa yaml[line-length]
+  cl/mnemonics.yaml: "{{ lookup('ansible.builtin.url', 'https://raw.githubusercontent.com/ethpandaops/ethereum-genesis-generator/v{{ethereum_genesis_generator_version}}/config-example/cl/mnemonics.yaml', split_lines=false) }}" # noqa yaml[line-length]
+  el/genesis-config.yaml: |
+    chain_id: ${CHAIN_ID}
+    deposit_contract_address: "${DEPOSIT_CONTRACT_ADDRESS}"
+    mnemonic: ${EL_AND_CL_MNEMONIC}
+    el_premine:
+      "m/44'/60'/0'/0/0": 10000000000ETH
```
Comment (parithosh) [inline ansible/inventories/devnet-10/group_vars/all/all.yaml:58]: would reduce each account by a bit so we dont risk overflow. perhaps like 1B? 
And we can do 20 accounts instead so we def avoid the risk?

File: ansible/inventories/devnet-10/group_vars/all/all.yaml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    export DENEB_FORK_VERSION="0x50{{ ethereum_genesis_fork_version_suffix }}"
+    export DENEB_FORK_EPOCH="256"
+    export WITHDRAWAL_TYPE="0x00"
+    export WITHDRAWAL_ADDRESS={{ ethereum_genesis_validator_bls_change_execution_address }}
+    export BEACON_STATIC_ENR=""
+    export GENESIS_TIMESTAMP={{ ethereum_genesis_timestamp }}
+    export GENESIS_DELAY={{ ethereum_genesis_timedelay }}
+    export SLOT_DURATION_IN_SECONDS=12
+    export MAX_CHURN=4
+    export EJECTION_BALANCE=31000000000
```
Comment (parithosh) [inline ansible/inventories/devnet-10/group_vars/all/all.yaml:114]: didnt you want to set it down to 16 again?

File: ansible/inventories/devnet-10/group_vars/all/all.yaml (hunk 1 above)
Comment (barnabasbusa) [reply ansible/inventories/devnet-10/group_vars/all/all.yaml:58]: If we catch overflow, I'll take that as a win. Will only reduce if we do catch an overflow tho :D

File: ansible/inventories/devnet-10/group_vars/all/all.yaml (hunk 2 above)
Comment (barnabasbusa) [reply ansible/inventories/devnet-10/group_vars/all/all.yaml:114]: do we ?

File: ansible/inventories/devnet-10/group_vars/tx_fuzz_blobs.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+tx_fuzz_blobs_container_image: "{{ default_tooling_images.tx_fuzz }}"
+tx_fuzz_blobs_container_name: tx_fuzz_blobs
+
+
+tx_fuzz_blobs_container_command:
+  - blobs
+  - --no-al
+  - --rpc=https://{{ secret_nginx_shared_basic_auth.name }}:{{ secret_nginx_shared_basic_auth.password }}@rpc.bootnode-1.srv.dencun-devnet-9.ethpandaops.io
```
Comment (parithosh) [inline ansible/inventories/devnet-10/group_vars/tx_fuzz_blobs.yaml:8]: fix URL. not devnet-9

File: ansible/inventories/devnet-10/group_vars/tx_fuzz_txs.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+tx_fuzz_txs_container_image: "{{ default_tooling_images.tx_fuzz }}"
+tx_fuzz_txs_container_name: tx_fuzz_txs
+
+tx_fuzz_txs_container_command:
+  - spam
+  - --no-al
+  - --rpc=https://{{ secret_nginx_shared_basic_auth.name }}:{{ secret_nginx_shared_basic_auth.password }}@rpc.bootnode-1.srv.dencun-devnet-9.ethpandaops.io
```
Comment (parithosh) [inline ansible/inventories/devnet-10/group_vars/tx_fuzz_txs.yaml:7]: fix URL

File: ansible/inventories/devnet-10/host_vars/mev-relay-1.yaml [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+  - --ws
+  - --ws.api=admin,engine,net,eth,web3,debug,flashbots
+  - --ws.addr=0.0.0.0
+  - --ws.port=8545
+  - --ws.origins=*
+  - --networkid={{ ethereum_network_id }}
+  - --syncmode=full
+  - --bootnodes={{ ethereum_el_bootnodes | join(',') }}
+  - --ethstats={{ inventory_hostname }}:{{ ethstats_secret }}@{{ ethstats_url }}
+  - --builder.remote_relay_endpoint=http://mev-api.dencun-devnet-9.ethpandaops.io:9062
```
Comment (parithosh) [inline ansible/inventories/devnet-10/host_vars/mev-relay-1.yaml:164]: fix URL

File: ansible/inventories/devnet-10/host_vars/mev-relay-1.yaml [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+  - --builder.beacon_endpoints=http://beacon:5052
+  - --builder.bellatrix_fork_version=0x30699478
```
Comment (parithosh) [inline ansible/inventories/devnet-10/host_vars/mev-relay-1.yaml:166]: fix genesis values if changed and val root cuz that's probably different

File: ansible/inventories/devnet-10/group_vars/all/all.yaml (hunk 1 above)
Comment (parithosh) [reply ansible/inventories/devnet-10/group_vars/all/all.yaml:58]: but then its a ton of wasted time for no reason no?
cuz the scenario can never happen on mainnet, inflation isnt high enough there

File: ansible/inventories/devnet-10/group_vars/all/all.yaml (hunk 2 above)
Comment (parithosh) [reply ansible/inventories/devnet-10/group_vars/all/all.yaml:114]: fine with or without, just thought of pointing it out

**Review by parithosh — APPROVED** [review]

**Comment by barnabasbusa** [comment]
This still uses old kzg trusted file! Rerun when you have the new kzg file!

**Pull request merged**